src/main/ts/common/LibraryUtils.ts

```diff
@@ -267,4 +267,9 @@ export default class LibraryUtils {
     if (/^[A-Z]:/.test(path) && path.indexOf("file://") == -1) path = "file://" + path; // prepend e.g. C: paths with file://
     return path;
   }
+
+  static readonly ThreadPool = new ThreadPool(1);
```

Comment:
Can rename to `MUTEX` to avoid using the class name and move to the very top alongside the other static variables.